packages/compiler/init/init.ts

```diff
@@ -37,8 +37,19 @@ interface ScaffoldingConfig extends InitTemplate {
    * Custom parameters provided in the tempalates.
    */
   parameters: Record<string, any>;
+
+  /**
+   * NormalizeVersion function replaces `-` with `_`.
+   */
+  normalizeVersion: any;
```

Comment:
Shouldn't have an any here. Looks like this can just be `() => string`?

Reply:
Actually will be func => func.  `() => (text: string, render: any) => string;` 
Fixed.